website/docs/guides/bigquery-qs.md

```diff
@@ -295,16 +439,80 @@ Later, you can connect your business intelligence (BI) tools to these views and
 
     This time, when you performed a `dbt run`, separate views/tables were created for `stg_customers`, `stg_orders` and `customers`. dbt inferred the order to run these models. Because `customers` depends on `stg_customers` and `stg_orders`, dbt builds `customers` last. You do not need to explicitly define these dependencies.
 
-
 #### FAQs {#faq-2}
 
 <FAQ path="Runs/run-one-model" />
 <FAQ path="Project/unique-resource-names" />
 <FAQ path="Project/structure-a-project" alt_header="As I create more models, how should I keep my project organized? What should I name my models?" />
 
-</div>
+## Build models on top of sources
+
+Sources make it possible to name and describe the data loaded into your warehouse by your extract and load tools. By declaring these tables as sources in dbt, you can:
+- select from source tables in your models using the `{{ source() }}` function, helping define the lineage of your data
+- test your assumptions about your source data
+- calculate the freshness of your source data
+
+1. Create a new YML file `models/sources.yml`.
+2. Declare the sources by copying the following into the file and clicking **Save**.
+
+    <File name='models/sources.yml'>
+
+    ```yml
+    version: 2
+
+    sources:
+        - name: jaffle_shop
+          description: This is a replica of the Postgres database used by our app
+          database: raw
+          schema: jaffle_shop
+          tables:
+              - name: customers
+                description: One record per customer.
+              - name: orders
+                description: One record per order. Includes cancelled and deleted orders.
+    ```
+
+    </File>
+
+3. Edit the `models/stg_customers.sql` file to select from the `customers` table in the `jaffle_shop` source.
```

Comment:
"Edit" in this situation isn't a UI element so shouldn't be bolded. This might be an ok false positive though and the author just has to ignore it 🤷‍♀️